Reference/Security/Security-hardening/index-v9.md

```diff
@@ -0,0 +1,84 @@
+---
+versionFrom: 9.0.0
+---
+
+# Umbraco Security Hardening
+
+Here you find some tips and trick for hardening the security of your Umbraco installation.
+
+## Lock down access to your Umbraco-folders (IIS)
+
+By default there are some folders in your Umbraco installation that should be only used by authenticated users. It’s considered a good practice to lock down these folders to specific IP-addresses and/or IP-ranges to make sure not everyone can access these.
+The folders we want to lock down are App_Plugins, Config and Umbraco.
+The prerequisite of this to work is that you’re using [IISRewrite](../../Routing/IISRewriteRules/index.md)
+
+If you’ve made sure that you’ve installed this on your server we can start locking down our Umbraco-folders. This can be down by following these three steps.
+
+1. We are going to lock down /Umbraco/, but because API-controllers and Surface-controller will use the path /umbraco/api/ and /umbraco/surface/ these will also be locked down. Our first rule in the IISRewrite.config will be used to make sure that these are not locked by IP-address.
```

Comment:
```suggestion
1. We are going to lock down `/Umbraco/`, but because API-controllers and Surface-controller will use the path `/umbraco/api/` and `/umbraco/surface/` these will also be locked down. Our first rule in the `IISRewrite.config` file will be used to make sure that these are not locked by IP address.
```